apps/front/src/components/company/CompanyForm.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <form @submit.prevent="$emit('submit-company', { ...form, identityFile })">
+    <div>
+      <label for="email">Name</label>
+      <input v-model="form.name" type="text" />
+    </div>
+    <div>
+      <label for="profilePicture">Identity file</label>
+      <input type="file" accept="image/*,.pdf" @input="onSelectFile" />
+    </div>
+    <button type="submit">Submit</button>
+  </form>
+</template>
+
+<script setup lang="ts">
+import { PropType } from "vue";
+import { Company } from "~/types/Company";
+
+const props = defineProps({
+  company: {
```

Comment:
+ we should use typescript props
+ Add more generic name so we can copy/paste forms